dot/parachain/backing/second.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2024 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package backing
+
+import (
+	"errors"
+	"fmt"
+
+	parachaintypes "github.com/ChainSafe/gossamer/dot/parachain/types"
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/ChainSafe/gossamer/pkg/scale"
+)
+
+var (
+	errWrongPVDForSecondingCandidate = errors.New(
+		"candidate backing was asked to second candidate with wrong persisted validation data")
+	errUnknownRelayParentForSecondingCandidate = errors.New(
+		"we were asked to second a candidate outside of our view")
+	errParaOutsideAssignmentForSeconding = errors.New(
+		"subsystem asked to second for parachain outside of our assignment")
+	errAlreadySignedValidStatement = errors.New("we have already signed a valid statement for this candidate")
```

Comment:
nit:

Is this error message copied from polkadot?

Reply:
Yes, the error messages are copied from Polkadot.

Reply:
I see. I kinda look at the node as a singular entity, so I'd suggest to remove "we". 

I'm just curious as to what the rest of the team thinks and feel free to resolve this before merging if you're in a hurry!